src/exampleTypes/ResultTable/TableBody.js

```diff
@@ -35,14 +35,15 @@ let TableBody = ({
               {...{ fields, visibleFields, hiddenFields }}
             >
               {_.map(
-                ({ field, display = x => x, Cell = Td }) => (
+                ({ field, display = x => x, Cell = Td, cellProps }) => (
```

Comment:
The idea of overriding `Cell` was so we wouldn't need to allow cellProps to be overridden since you can just override Cell and add whatever props you wanted. I'm not completely opposed to adding it, but I wonder if it's needed

Reply:
I'm +1 on `cellProps` since `defaultProps` creates a wrapper. Same argument as autoform really. It's more convenient to have `props` than `component: defaultProps(...)(Input)`

Reply:
I've actually already removed `cellProps` in favor of `Cell`. Just to keep the props footprint simpler. Having `Cell: defaultProps(...)(Td)` makes code only couple chars longer in this case